kuma/javascript/src/document-provider.jsx

```diff
@@ -67,7 +67,7 @@ export default function DocumentProvider(
         // This is the function that does client side navigation
         function navigate(url, localeAndSlug) {
             body.style.opacity = '0.15';
-            fetch(`/api/v1/doc${localeAndSlug}`)
+            fetch(`/api/v1/doc${localeAndSlug}`, {redirect:'follow'})
```

Comment:
MDN says that redirect:follow is the default, but implies that it might only be the default on Chrome? So this might not be necessary, but it seems safer to make it explicit, as you have done.